force-app/main/default/lwc/givingSummary/givingSummary.js

```diff
@@ -16,16 +16,20 @@ const FormFactorType = Object.freeze({
     Small: 'Small',
 });
 
-const MOBILE_CLASSES = 'slds-text-align_center slds-border_bottom slds-text-heading_small slds-var-p-around_medium';
-const DESKTOP_CLASSES = 'slds-text-align_center slds-border_right slds-text-heading_small';
+const MOBILE_CLASSES = 'slds-text-align_left slds-border_right columns-mobile slds-var-p-around_small';
+const DESKTOP_CLASSES = 'slds-text-align_left slds-border_right slds-text-heading_small slds-p-left_medium';
 const FIELDS = [TOTAL_AMOUNT, AMOUNT_CURRENT_YEAR, AMOUNT_LAST_YEAR];
 
+const labelThisYear = donationHistoryLabelThisYear.toUpperCase();
```

Comment:
Could also use `text-transform: uppercase;` in the CSS 🤷‍♂️  up to you!

Reply:
It's a good suggestion, I'll change it